starter-code/js/index.js

```diff
@@ -1 +1,57 @@
-// Start here :)
\ No newline at end of file
+// Start here :)
+
+// const calculateButton = document.getElementById("calculate");
+// const produits = document.getElementsByClassName("name");
+calcButton = document.getElementById("calculate")
+
+calcButton.onclick = function() {
+    console.log("click");
+    var $allProducts = [...document.querySelectorAll(".products")];
+    var globalPrice = 0;
+    for (const product of $allProducts) {
+        let couts = product.querySelector('.cost span').innerHTML;
```

Comment:
Ici, utiliser `for...of` n'est pas nécessaire pour looper sur vos produits, car vous les avez préalablement spreadés dans un array :
```javascript
var $allProducts = [...document.querySelectorAll(".products")];
```

Vous pouvez donc utiliser un `forEach` sans souci :)